docs/source/config/webui.rst

```diff
@@ -58,7 +58,7 @@ On an |st2| side, `CORS <https://en.wikipedia.org/wiki/Cross-origin_resource_sha
    logging = st2api/conf/logging.conf
    # List of allowed origins for CORS, use when deploying st2web client
    # The URL must match the one the browser uses to access the st2web
-   allow_origin = http://webui.example.com:8000
+   allow_origin = http://webui.example.com
```

Comment:
So it depends on a URL you are using to access the UI. In case of AIO, you don't need to edit it at all.